superset-frontend/src/common/components/Tabs.tsx

```diff
@@ -16,27 +16,46 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import React from 'react';
 import { styled } from '@superset-ui/core';
 import { Tabs as AntdTabs } from 'src/common/components';
+import { css } from '@emotion/core';
```

Comment:
You can import this from @superset-ui/core along with styled a couple lines up.

Reply:
Done